src/web/mjs/connectors/KissAway.mjs

```diff
@@ -14,17 +14,12 @@ export default class KissAway extends FlatManga {
     async _getChapters(manga) {
         const script = `
             new Promise(async resolve => {
-                const uri = new URL('app/manga/controllers/cont.listChapter.php', window.location.origin);
-                uri.searchParams.set('slug', dataL);
-                const response = await fetch(uri);
-                data = await response.text();
-                const dom = new DOMParser().parseFromString(data, "text/html");
-                const nodes = [...dom.querySelectorAll('a.chapter[title]')];
-                const chapters= nodes.map(chapter => {
+                const nodes = document.querySelectorAll("div#list-chapter a")
+                const chapters = Array.prototype.map.call(nodes, item => {
                     return {
-                        id : chapter.pathname,
-                        title : chapter.title
-                    };
+                        id : item.href,
```

Comment:
never use full url for identifiers unless absolute necessity. But yeah, there may be a typo here